### PR TITLE
adding email in admin-sales-organiser model

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This fixes the `admin-sales-organiser`  model and the corresponding template in the admin section.

#### Changes proposed in this pull request:

- fix the syntax error in the template
- Adding email to be shown in case the profile contains no name

image

Fixes fossasia/open-event-server#5621

### CODE REVIEW

**Comment by uds5501** [comment]
Corresponding server PR fossasia/open-event-server#5625

**Comment by uds5501** [comment]
@iamareebjamal @CosmicCoder96 @niranjan94 Please review

File: app/templates/admin/sales/organizers.hbs [hunk 1]
```diff
@@ -45,7 +45,13 @@
           {{#each model as |entry|}}
             <tr>
               <td>
-                {{entry.firstName entry.lastName}}
+                {{#if entry.firstName}}
+                  {{entry.firstName}} {{entry.lastName}}
+                {{else}}
+                  {{t 'No Name Provided'}}
+                {{/if}}
+                <br>
```
Comment (CosmicCoder96) [inline app/templates/admin/sales/organizers.hbs]: don't use `br` tag, use a div with class `muted text` instead for the email.

**Comment by uds5501** [comment]
@CosmicCoder96 suggestions acknowledged, please review now

**Pull request merged**